integration-tests/src/test/java/oracle/weblogic/kubernetes/ItFmwDynamicDomainInImage.java

```diff
@@ -0,0 +1,309 @@
+// Copyright (c) 2021, Oracle and/or its affiliates.
```

Comment:
Modify the class name to ItFmwDynamicClusterMiiDomain.java to best represent the usecae

Reply:
changed